README.md

```diff
@@ -2,39 +2,298 @@
-| `/field` [NOT IMPLEMENTED YET]                                            | `GET`    | Retrieves the field values of all API subscriptions                                            | `curl -v -X GET    "http://localhost:9650/field                                                                        -H "Cache-Control: no-cache"`                                                                                                                                         | 200 OK `{"subscriptions":[{"clientId":"hBnFo14C0y4SckYUbcoL2PbFA40a","apiContext":"ciao-api","apiVersion":"1.0","fieldsId":"f121ffa3-df94-43a0-8235-ac4530f9700a","fields":{"callback-url":"http://localhost:8080/callback","token":"abc59609za2q"}},{"clientId":"gGnFj14C05fScBYUbcoL3PbF540a","apiContext":"hello-api","apiVersion":"2.0","fieldsId":"ab1a03cf-8666-4e33-9ef7-07c172527675","fields":{"phone":"01219928876","city":"Leeds"}}]}`                 |
-| `/field/application/:client-id/context/:api-context/version/:api-version` | `DELETE` | Deletes the field values of an API subscription                                                | `curl -v -X DELETE "http://localhost:9650/field/application/hBnFo14C0y4SckYUbcoL2PbFA40a/context/ciao-api/version/1.0" -H "Cache-Control: no-cache"`                                                                                                                                         | 404 NOT_FOUND / 204 No Content                                                                                                                                                                                                                                                                                                                                                                                                                                    |
+## PUT Field Definitions 
+### `PUT /definition/context/:apiContext/version/:apiVersion`
+Creates or updates the definitions of the subscriptions fields for an API.
 
+### Response with
 
-## Tests
-Some tests require MongoDB to run. 
-Thus, remember to start up MongoDB if you want to run the tests locally.
-There are unit tests, integration tests and acceptance tests plus code coverage reports are generated too.
-In order to run them, use this command line:
+| Status | Description                          |
+|--------|--------------------------------------|
+| 201    | Created                              |
+| 200    | Updated                              |
+| 400    | Bad request                          |
```

Comment:
I prefer to mention only the codes we return in our code, but not the 5xx.
So, I would mention only 200, 201, 204 and 404.
I prefer we do not mention the other codes.